src/pool/srv_pool.c

```diff
@@ -6989,8 +6987,9 @@ pool_svc_update_map_internal(struct pool_svc *svc, unsigned int opc,
 		}
 
 		failed_cnt = pool_map_get_failed_cnt(map, PO_COMP_TP_NODE);
-		D_INFO(DF_UUID": SWIM exclude %d ranks, failed NODE %d\n",
-		       DP_UUID(svc->ps_uuid), tgt_addrs->pta_number, failed_cnt);
+		if (src == MUS_SWIM)
+			D_INFO(DF_UUID ": SWIM exclude %d ranks, failed NODE %d\n",
+			       DP_UUID(svc->ps_uuid), tgt_addrs->pta_number, failed_cnt);
```

Comment:
Should we remove "SWIM" from the log message instead? For instance, "exclude %d ranks: %d failed nodes\n".